types/systemjs/index.d.ts

```diff
@@ -1,11 +1,26 @@
 // Type definitions for SystemJS 0.20
 // Project: https://github.com/systemjs/systemjs
-// Definitions by: Ludovic HENIN <https://github.com/ludohenin>, Nathan Walker <https://github.com/NathanWalker>, Giedrius Grabauskas <https://github.com/GiedriusGrabauskas>
+// Definitions by: Ludovic HENIN <https://github.com/ludohenin>, Nathan Walker <https://github.com/NathanWalker>, Giedrius Grabauskas <https://github.com/GiedriusGrabauskas>,
+//                 Aluan Haddad <https://github.com/aluanhaddad>
```

Comment:
Split to new lines.
https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package